packages/vsphere-cpi/charts/values.yaml

```diff
@@ -0,0 +1,20 @@
+vCenter:
+  host: ""
+  port: 443
+  insecureFlag: "1"
```

Comment:
I swear this was true a second ago.

Reply:
it was, we reverted it

Reply:
https://github.com/rancher/helm3-charts/pull/62#discussion_r586912945

Reply:
I think it was other PR